### DISCUSSION TITLE
[Auth] Allow custom logos

### DISCUSSION THREAD

**Discussion by Sytten** [ente-io/ente#1859] [Enhancements]
Unsure if this was discussed but it is missing a lot of logos and some I don't expect you guys to ever support natively. Still coming from Authy I am used to browsing my codes and seeing the logos.
Ideally I would like to be able to upload my own SVG for a given entry.

**Comment by sv3nnie** [comment]
You can contribute custom icons, see [this](https://github.com/ente-io/ente/blob/main/auth/docs/adding-icons.md) readme. After contributing, everyone will be able to use them.
